### PR TITLE
DEVX-1881: Update microservices code to work with secure services

### PR DESCRIPTION
**Pull request by rspurgeon**
DEVX-1881: Update microservices code to work with secure clusters and SR.  Configuration values can be passed in with properties files to the various services.

The microservices have been modified to accept CLI named arguments instead of positional arguments.

### CODE REVIEW

**Comment by rspurgeon** [comment]
[clabot:check]

File: src/main/java/io/confluent/examples/streams/microservices/AddInventory.java [hunk 1]
```diff
@@ -46,19 +52,50 @@ private static void sendInventory(final List<KeyValue<Product, Integer>> invento
         }
     }
 
-    public static void main(final String[] args) {
+    public static void main(final String[] args) throws Exception {
+
+        final Options opts = new Options();
+        opts.addOption(Option.builder("b")
+                .longOpt("bootstrap-server").hasArg().desc("Kafka cluster bootstrap server string").build())
```
Comment (ybyzek) [inline src/main/java/io/confluent/examples/streams/microservices/AddInventory.java:59]: @rspurgeon for consistency, consider making this plural `servers`

Comment (rspurgeon) [reply src/main/java/io/confluent/examples/streams/microservices/AddInventory.java]: done

**Comment by guozhangwang** [comment]
LGTM.

**Pull request merged**